src/service/viewport/viewport-impl.js

```diff
@@ -210,6 +210,11 @@ export class Viewport {
       globalDocElement.classList.add('i-amphtml-webview');
     }
 
+    // Doc Level CSS Experiments
+    if (isExperimentOn(this.ampdoc.win, 'inabox-preserve-height-auto')) {
```

Comment:
for easier rollout, better do the reverse way. 0 -> 100%
also, you will need to add the flag to the list for `allow-doc-opt-in` in prod-config.json so that the experiment can be fully controlled server side.